CHANGELOG.md

```diff
@@ -48,6 +48,7 @@
 
 ### Code Cleanup
 
+- All references to Git submodules removed since there are no more.
```

Comment:
This should be added in the CHANGELOG for the next release. It will be automatically added when this PR gets merged.

Reply:
Fixed.